src/vuln_analysis/functions/exploitability_analyzer.py

```
@@ -451,7 +451,8 @@ def _extract_cve_id(vulnerability_report_path: str) -> str:
         match = re.search(r"(CVE-\d{4}-\d+)", content)
         if match:
             return match.group(1)
-    except Exception:
+    # Unreadable reports fall back to the path-derived identifier.
+    except Exception:  # nosec B110
         pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)